Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -730,6 +801,66 @@ def panorama_push_to_device_group_command(args: dict):
         return_results(result['response']['msg']['line'])
 
 
+def panorama_push_to_template_command(args: dict):
+    """
+    Push Panorama Template to it's associated firewalls
+    """
+
+    if not TEMPLATE:
+        raise Exception("The 'panorama-push-to-template' command is relevant for a Palo Alto Panorama instance.")
+
+    result = panorama_push_to_template(args)
+    if 'result' in result['response']:
+        # commit has been given a jobid
+        push_output = {
+            'Template': TEMPLATE,
+            'JobID': result['response']['result']['job'],
+            'Status': 'Pending'
+        }
+        return_results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': result,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('Push to Template:', push_output, ['JobID', 'Status'],
+                                             removeNull=True),
+            'EntryContext': {
+                "Panorama.Push(val.JobID == obj.JobID)": push_output
+            }
```

Comment:
is there a reason you don't use ```CommandResults```? or is it mainly because that is the way its implemented in most of the integration? I'd be happy to see you using ```CommandResults``` instead

Reply:
I think it's a little simpler to keep each of these functions looking the same as they are all really similar